nix/bazel.nix

```diff
@@ -51,10 +51,7 @@ let shared = rec {
   # created with bundix.
   sass = pkgs.callPackage ./overrides/sass {};
 
-  sphinx183 = import ./tools/sphinx183 {
-    inherit pkgs;
-    pythonPackages = pkgs.python37Packages;
-  };
+  sphinx183 = pkgs.python3Packages.sphinx;
```

Comment:
Now that we are on `nixpkgs-unstable` I don’t see a reason to keep our own vendored version of `sphinx` since we get updates fairly quickly. If we do end up getting out of sync, we can always go back.